src/utils/parseTxData/api.ts

```diff
@@ -21,7 +21,9 @@ export const api = async (address: string) => {
     res[jettonInfo.symbol] = {
       ...jettonInfo,
       pnlPercentage,
-      chart: (slicedChart.length >= 2 ? slicedChart : chart).reverse().map(entity => [entity[0], normalizePrice(entity[1], jettonInfo.decimals)]),
+      chart: (slicedChart.length >= 2 ? slicedChart : chart)
```

Comment:
If `slicedChart` does not have at least two elements, that means our algorithm is wrong. log it as error.

Reply:
Do not reverse the chart here. Reverse it on the frontend

Reply:
I just didn't touch that, but sure.